src/installer.ts

```diff
@@ -170,6 +175,36 @@ export async function getNode(
   core.addPath(toolPath);
 }
 
+function isLts(versionSpec: string): boolean {
+  return versionSpec.startsWith('lts')
+}
+
+function findLtsVersionFromManifest(
+  versionSpec: string,
+  stable: boolean,
+  candidates: Array<tc.IToolRelease & { lts?: string }>
+): string {
+  const alias = versionSpec.split('lts/')[1]?.toLowerCase();
+
+  if (!alias) {
+    throw new Error(`Unexpected LTS alias '${alias}' for Node version '${versionSpec}'`);
+  }
+
+  core.debug(`LTS alias '${alias}' for Node version '${versionSpec}'`);
+
+  const release = alias === '*'
+   ? candidates.find(x => !!x.lts && x.stable === stable)
+   : candidates.find(x => x.lts?.toLowerCase() === alias && x.stable === stable);
+
+  if (!release) {
+    throw new Error(`Unable to find LTS release '${alias}' for Node version '${versionSpec}'.`);
+  }
+
+  core.debug(`Found LTS release '${release.version}' for Node version '${versionSpec}'`);
+
+  return release.version.split('.')[0];
```

Comment:
Why do we need to return major version only? What will happen if we will return exact version? 
I think `resolveVersionFromManifest` will handle it correctly.